src/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+
+type ButtonProps = {
+  name: string;
+  content: string;
+  filterHandler: (action: string) => void;
+  filter: string;
+  isReversed: boolean;
+};
+
+export const Button: React.FC<ButtonProps> = ({
+  name,
+  content,
+  filterHandler,
+  filter,
+  isReversed,
+}) => {
+  const buttonClass = classNames('button', {
+    'is-info': name === 'alphabet',
+    'is-success': name === 'length',
+    'is-warning': name === 'reverse',
+    'is-danger': name === 'reset',
+    'is-light': name !== filter && !(name === 'reverse' && isReversed),
```

Comment:
[UX] - The current implementation does not visually indicate which button is active. Following the checklist, you should mark the active button to explain to the user what action is active. Consider adding a class that changes the appearance of the button when it corresponds to the current filter or when the reverse action is active. For instance, you can add a 'is-active' class when `name === filter` or when `name === 'reverse' && isReversed` is true.